unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Scripts/SectionController/MenuSections/SectionLandController.cs

```diff
@@ -0,0 +1,129 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using Object = UnityEngine.Object;
+
+internal class SectionLandController : SectionBase, ILandsListener
+{
+    public const string VIEW_PREFAB_PATH = "BuilderProjectsPanelMenuSections/SectionLandsView";
+    
+    public override ISectionSearchHandler searchHandler => landSearchHandler;
+
+    private readonly SectionLandView view;
+
+    private readonly LandSearchHandler landSearchHandler = new LandSearchHandler();
+    private readonly Dictionary<string, LandElementView> landElementViews = new Dictionary<string, LandElementView>();
+    private readonly Queue<LandElementView> landElementViewsPool = new Queue<LandElementView>();
+
+    public SectionLandController() : this(
+        Object.Instantiate(Resources.Load<SectionLandView>(VIEW_PREFAB_PATH))
+        )
+    {
+    }
+    
+    public SectionLandController(SectionLandView view)
+    {
+        this.view = view;
+        PoolView(view.GetLandElementeBaseView());
+
+        landSearchHandler.OnResult += OnSearchResult;
+    }
+
+    public override void SetViewContainer(Transform viewContainer)
+    {
+        view.SetParent(viewContainer);
+    }
+
+    public override void Dispose()
+    {
+        view.Dispose();
+    }
+
+    protected override void OnShow()
+    {
+        view.SetActive(true);
+    }
+
+    protected override void OnHide()
+    {
+        view.SetActive(false);
+    }
+
+    void ILandsListener.OnSetLands(LandData[] lands)
+    {
+        view.SetEmpty(lands.Length == 0);
+
+        List<LandElementView> toRemove = new List<LandElementView>(
+            landElementViews.Values.Where(landElementView => lands.All(land => land.id != landElementView.GetId())));
+        
+        for (int i=0; i<toRemove.Count; i++)
```

Comment:
This document is not following our code style

Reply:
fixed code formatting